src/types.ts

```diff
@@ -197,3 +199,69 @@ export interface HistoryLookup {
 export const HistoryLookupSpec: ObjectSpec<HistoryLookup> = {
   bugs: array(object(BugHistorySpec)),
 };
+
+export interface Comment {
+  attachment_id?: int | null;
+  bug_id: int;
+  count: int;
+  creation_time: datetime;
+  creator: string;
+  id: int;
+  // is_markdown: boolean;
```

Comment:
Strangely `is_markdown` property is documented in [docs](https://bugzilla.readthedocs.io/en/latest/api/core/v1/comment.html#get-comments), but for some reason it is missing from output.

Example:
```json
{
  "creation_time": "2015-06-22T17:53:39Z",
  "tags": [],
  "creator": "jonathan.underwood",
  "text": "The Emacs add-on packaging guidelines no longer stipulate that packages which also bundle support for Emacs should split out those Emacs files into separate sub-packages. This package should instead ship those files with the main package which should also Require emacs-filesystem. See https://fedoraproject.org/wiki/Packaging:Emacs for more detail.",
  "time": "2015-06-22T17:53:39Z",
  "count": 0,
  "bug_id": 1234567,
  "is_private": false,
  "id": 8360743,
  "creator_id": 163554,
  "attachment_id": null
}
```

@Mossop, Could it be obsolete documentation?

Reply:
I guess so. The REST API docs seem to be incorrect in a number of places unfortunately.